DittoHealthMetrics/build.gradle

```
@@ -11,7 +11,7 @@ apply from: "$rootProject.projectDir/gradle/deploy.gradle"
 apply from: "$rootProject.projectDir/gradle/android-common.gradle"
 
 android {
-    namespace "live.ditto.health"
+    namespace "live.ditto.dittohealthmetrics"
```

I got the impression from @texasRanger09's [comment](https://github.com/getditto/DittoAndroidTools/pull/77#discussion_r1613727266) that this was changing to `live.ditto.healthmetrics`? Is that still desired?

```suggestion
    namespace "live.ditto.healthmetrics"
```

---

Since this discussion originally happened, we've published this as `live.ditto.health`: https://central.sonatype.com/artifact/live.ditto/health.

Not sure if that changes what we want to do here?

---

...which seems to have resulted in https://github.com/getditto/DittoAndroidTools/issues/101 since we now have two `live.ditto.health` modules based on what @rajramsaroop found today